azalea-auth/src/auth.rs

```diff
@@ -24,6 +24,10 @@ pub struct AuthOpts {
     /// The directory to store the cache in. If this is not set, caching is not
     /// done.
     pub cache_file: Option<PathBuf>,
+    /// If you choose to use your own microsoft authentication instead of using nintendo switch, just put your client_id here
```

Comment:
Tiny nitpick here but I've been trying to keep the grammar in the docs consistent, mainly just capitalizing proper nouns (so Microsoft and Nintendo Switch should be capitalized here) and adding periods at the end of sentences.